src/annotation.js

```diff
@@ -165,13 +164,13 @@ function drawElements(ctx, elements, caller) {
     if (el.options.drawTime === caller) {
       el.draw(ctx);
     }
-  }
-}
-
-function drawSubElements(ctx, elements, caller) {
-  for (const el of elements) {
-    if (isArray(el.elements)) {
-      drawElements(ctx, el.elements, caller);
+    if (el.elements && el.elements.length) {
+      const box = 'getBoundingBox' in el ? el.getBoundingBox() : undefined;
```

Comment:
@kurkle maybe you don't like it. I prefer to pass an undefined value instead of a default. Let me know.